kustomize/components/google-cloud-operations/otel-collector.yaml

```diff
@@ -37,16 +37,9 @@ spec:
       # https://cloud.google.com/compute/docs/storing-retrieving-metadata
       initContainers:
       - name: otel-gateway-init
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-              - all
-          privileged: false
-          readOnlyRootFilesystem: true
```

Comment:
**Question:**
Is there a reason we removed this:
```
        securityContext:
          allowPrivilegeEscalation: false
          capabilities:
            drop:
              - all
          privileged: false
          readOnlyRootFilesystem: true
```

Reply:
I think that's a victim of a merge gone wrong.  I've fixed it.